src/liberty/devCommands.ts

```diff
@@ -610,4 +611,32 @@ async function getLocalGradleWrapper(projectFolder: string): Promise<string | un
  */
 function isWin(): boolean {
     return process.platform.startsWith("win");
-}
\ No newline at end of file
+}
+
+function getReportFile(path :any,dir :string,filename:string):any{
+    console.log("inside getReportFile path::"+path+"::dir::"+dir+"::fileName::"+filename);
+    console.log("::path::"+Path.join(path,"target",dir,filename));
+    return Path.join(path,"target",dir, filename);
+}
+
+/*
+Function will check if the report is available within the given path and returns a boolean based on it and also 
+  the report will be displayed if it is available
+*/
+function checkReportExists(report : any,reportType : string,reportTypeLabel: string,libProject :LibertyProject): Promise<boolean> {
```

Comment:
Please rename this method, as it not only checks for the report's existence but also opens it in a web view.

Reply:
or add a new method for displaying report and use this method only to check report exist or not.